load_blazegraph_test.sh

```diff
@@ -10,16 +10,85 @@ BASEURI="http://ld4p-test.stanford.edu/"
 # vars above this line may need to change to process other data
 #------------------------------------------------
 
-BG_URL="http://localhost/blazegraph/"
-TEST_NAMESPACE=test
```

Comment:
To enable an automated creation or check that the `test` namespace exists, we would need an additional script; e.g. see SO commentary about how this is done at
- http://stackoverflow.com/questions/41300401/blazegraph-create-namespace-via-rest-api

Are you recommending that we do this instead of using the `kb` namespace that is always created by blazegraph?